raster-foundry/raster-foundry
# Add backend support for project layer async exports

## Overview

This PR:
 - adds a `project_layer_id ` field to `exports` table
 - adds backend support for project layer async exports

### Checklist

- ~Styleguide updated, if necessary~
- ~[Swagger specification](https://github.com/raster-foundry/raster-foundry-api-spec) updated~
- ~Any content changes are properly templated using `BUILDCONFIG.APP_NAME`~
- ~Any new SQL strings have tests~

## Testing Instructions

 * Run migrations
 * Go to a project and add a scene to it
 * On project export UI, create two exports
 * On new project layer list UI (`/v2/project/<projectId>`), create a new layer in addition to the default layer
 * Clone the default layer's `scenes_to_layers` record, and substitute the layer id with the layer created in step 4
 * Update one of the export records created as a result of step 3, set its `project_layer_id` with that of step 4
 * `backsplash-export/assembly`
 * `batch/assembly`
 * `docker-compose build batch`
 * `./scripts/console batch bash`
 * Make sure both `rf export <exportIdOne>` and `rf export <exportIdTwo>` go through
 * From the old project export UI, check the exports, download them, view them in a GIS software

Closes https://github.com/raster-foundry/raster-foundry/issues/4486
